build.gradle

```diff
@@ -102,6 +102,9 @@ dependencies {
     testImplementation "org.apache.kafka:connect-json:$kafkaVersion"
     testImplementation "org.junit.jupiter:junit-jupiter:$junitVersion"
     testImplementation "com.fasterxml.jackson.core:jackson-databind:2.9.4"
```

Comment:
is that a reaon why we use old version of `jackson-databind`? `2.11.3` has been released

Reply:
Lets update!
- [update version]